sunpy/database/tests/test_tables.py

```diff
@@ -248,7 +248,8 @@ def test_entry_from_qr_block_kev(qr_block_with_kev_unit):
     assert entry.provider == 'LSSP'
     # TODO: Flaky test that needs a fix
     assert entry.fileid in ['/hessidata/2011/09/19/hsi_20110919_233340',
-                            '/hessidata/2011/09/20/hsi_20110920_010920']
+                            '/hessidata/2011/09/20/hsi_20110920_010920',
+                            '/hessidata/2011/09/19/hsi_20110919_233340_002.fits']
```

Comment:
I confirmed with Richard Schwartz, Kim Tolbert and Jim McTiernann that these files are unlikely to change again anytime soon.